docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3551,6 +3551,11 @@ Remove a node [`id`] from the Swarm.
     Content-Length: 0
     Content-Type: text/plain; charset=utf-8
 
+**Query parameters**:
+
+-   **force** - 1/True/true or 0/False/false, Force remove an active node.
```

Comment:
note: not sure if we should say `worker node`. you still have to demote manager nodes before this will work.